api/src/database/db.test.ts

```diff
@@ -114,17 +115,24 @@ describe('db', () => {
           it('throws an error', async () => {
             const getDBPoolStub = sinonSandbox.stub(db, 'getDBPool').returns(undefined);
 
-            let expectedError: Error;
+            let expectedError: ApiExecuteSQLError;
             try {
               await connection.open();
 
               expect.fail('Expected an error to be thrown');
             } catch (error) {
-              expectedError = error as Error;
+              expectedError = error as ApiExecuteSQLError;
             }
 
-            expect(expectedError.message).to.equal('DBPool is not initialized');
+            expect(expectedError.message).to.equal('Failed to execute SQL');
 
+            expect(expectedError.errors?.length).to.be.greaterThan(0);
+            expectedError.errors?.forEach((item) => {
+              expect(item instanceof Error).to.be.true;
```

Comment:
nit: can we change this to use .instanceof? expect(blah).to.be.instanceof(stuff)

Just to match the other tests, though I think there is one other test file that also does it this way and could be updated.